mmdeploy/backend/tensorrt/torch_allocator.py

```diff
@@ -53,11 +53,10 @@ def deallocate(self: trt.IGpuAllocator, memory: int) -> bool:
         Returns:
             bool: deallocate success.
         """
-        logger = get_root_logger()
```

Comment:
Can we still keep them if mmengine is installed?

Reply:
Yes. We can also modify the codes of [get_logger()](https://github.com/open-mmlab/mmdeploy/blob/master/mmdeploy/utils/logging.py).